property_caching/__init__.py

```diff
@@ -38,12 +45,14 @@ def _get_cache(obj, cache_attr_name=_OBJ_CACHE_ATTR_NAME):
     return getattr(obj, cache_attr_name, {})
 
 
-def _get_property_value(fn, obj, cache_attr_name):
+def _get_property_value(fn, obj, cache_attr_name, only_cache_true_vals=False):
+    result = None
```

Comment:
It should never get to 58 with this `None` (unless I'm missing something).  And if it is - we have some serious issue with our logic.  But I might be completely dumb :)